### ISSUE TITLE
How to run on Windows

### ISSUE THREAD

**Issue by take0212** [HRNet/HRNet-Semantic-Segmentation#157]
When I run the code on Windows, I get the attached error.
How can I solve it?
[06302020error_log_and_version_info.txt](https://github.com/HRNet/HRNet-Semantic-Segmentation/files/4849341/06302020error_log_and_version_info.txt)

**Comment by take0212** [comment]
I fixed the bug myself.
I confirmed that it runs on Windows.

Modified points are as follows.

- Change Inplace-ABN to standard BN.
- Disable distributed training.

(1). About changing Inplace-ABN to standard BN.
Inplace-ABN is not useful with a single GPU and  C++ source codes must be compiled in v0.4.1.
So I changed Inplace-ABN to standard BN.

v0.4.1
lib\models\seg_hrnet.py

`Change InPlaceABNSync to nn.BatchNorm2d.
`

v1.1
tools\train.py

`Delete nn.SyncBatchNorm.convert_sync_batchnorm.
`

(2).About disabling distributed training.
PyTorch doesn't support distributed training on Windows.
So I disabled distributed training.

v1.1
tools\train.py

`Change nn.parallel.DistributedDataParallel to nn.DataParallel.
`

(3).Other

`Adapte GPUS and WORKERS of yaml to my GPU PC.
`
